gallery/src/styles/NxFormLayout/NxFormHorizontalLayoutExample.tsx

```diff
@@ -47,6 +47,26 @@ export default function NxFormLayoutExample() {
           </label>
         </div>
       </div>
+      <div className="nx-form-row">
+        <div className="nx-form-group">
+          <label className="nx-label nx-label--optional">
+            <span className="nx-label__text">Label</span>
+            <NxStatefulTextInput/>
+          </label>
+        </div>
+        <div className="nx-form-group">
+          <label className="nx-label">
+            <span className="nx-label__text">Label</span>
+            <select className="nx-form-select">
```

Comment:
This one needs state management too

Reply:
> This one needs state management too

https://github.com/sonatype/sonatype-react-shared-components/pull/203/commits/a329a17b7cda35bbf29e06fca284f1b3ae6255ea